html/signupsignin.html

```diff
@@ -9,28 +9,14 @@
     <link rel="stylesheet" href="../css/signupsignin.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
         integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
-    <link rel="stylesheet" href="createprofile.css">
 </head>
 
 <body>
     <script src="https://www.gstatic.com/firebasejs/7.15.1/firebase-app.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.1/firebase-analytics.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.0/firebase-auth.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.0/firebase-firestore.js"></script>
-    <script>
-        const firebaseConfig = {
-            apiKey: "AIzaSyBbuvg-wCgz30zyD4zETU0HiHiC5_FulWc",
-            authDomain: "good-teens.firebaseapp.com",
-            databaseURL: "https://good-teens.firebaseio.com",
-            projectId: "good-teens",
-            storageBucket: "good-teens.appspot.com",
-            messagingSenderId: "260739514487",
-            appId: "1:260739514487:web:8197a36e539f090f2ec672",
-            measurementId: "G-0WBRPFKT5Z"
-        };
-        // Initialize Firebase
-        firebase.initializeApp(firebaseConfig);
-    </script>
+    <script src="../js/setupFirebase.js"></script>
     <div class="container">
         <div class="column1">
             <form action="#">
```

Comment:
this action is breaking the proper execution of signup (maybe signin as well). Please remove it.

Reply:
I removed the 'action' from the <form> completely, but it didn't help. It isn't yet working, a bug must be somewhere else. I will keep trying, will make another push once it works.